explorer/src/lib/components/statistics-panel/StatisticsPanel.css

```diff
@@ -0,0 +1,104 @@
+.statistics-panel {
+  display: flex;
+  padding: 1rem 1.375rem;
+  flex-direction: column;
+  column-gap: 0.625rem;
+  row-gap: 0.75rem;
```

Comment:
Is just `gap: 0.75rem;` not enough?

Reply:
No, as the column and row gaps are no equal.

Reply:
<img width="576" alt="Screenshot 2024-04-09 at 15 06 13" src="https://github.com/dusk-network/rusk/assets/34094428/4f1e7a1f-48c8-4feb-8908-0380f95a4733">

On mobile there is only one column, so `column-gap` does nothing. Unless I'm going crazy...

Reply:
Ah, yes, leftover from when i used a flexbox system instead.